src/Kunstmaan/NodeBundle/Tests/Entity/NodeTest.php

```diff
@@ -161,6 +170,36 @@ public function testToString()
         $this->assertEquals('node 1, refEntityName: Kunstmaan\NodeBundle\Tests\Entity\TestEntity', $this->object->__toString());
     }
 
+
+	public function testPageShouldHaveTab() {
+		$request = new Request();
+		$request->request = new ParameterBag();
+
+		$formFactory = $this->getMock(FormFactory::class, [], [], '', false);
```

Comment:
FYI, instead of a bunch parameters nobody knows the purpose for, you can write:

```php
$this->getMockBuilder(FormFactory::class)->disableOriginalConstructor()->getMock()
```